### PR TITLE
@lsetiawan suggestions

### PR DESCRIPTION
**Pull request by ocefpaf**
Addresses all @lsetiawan suggestions but https://github.com/BiG-CZ/notebook_data_demo/issues/3#issuecomment-313152426, which is pending @emiliom opinion.

@lsetiawan you have quite a talent for `css`! The page looks much better now.

### CODE REVIEW

**Comment by lsetiawan** [comment]
> @lsetiawan you have quite a talent for css! The page looks much better now.

Haha thanks 😅. I do some web design on the side. 😄

File: webpage/jupyter_notebook_jekyll_theme/_sass/_masthead.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
   -webkit-animation-delay: 0.15s;
           animation-delay: 0.15s;
   z-index: 20;
-  background-color: $masthead-background-color;
+  background-color: #5E5C57;
```
Comment (lsetiawan) [inline webpage/jupyter_notebook_jekyll_theme/_sass/_masthead.scss]: I'm surprised you didn't change this in https://github.com/BiG-CZ/notebook_data_demo/blob/master/webpage/jupyter_notebook_jekyll_theme/_sass/_variables.scss#L64

File: webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss [hunk 2]
```diff
@@ -175,7 +175,7 @@
 .greedy-nav {
   position: relative;
   min-width: 250px;
-  background-color: $greedy-nav-background-color;
+  background-color: #5E5C57;
```
Comment (lsetiawan) [inline webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss]: Same, not changing in https://github.com/BiG-CZ/notebook_data_demo/blob/master/webpage/jupyter_notebook_jekyll_theme/_sass/_variables.scss#L66?

**Review by lsetiawan — COMMENTED** [review]
Just some questions otherwise, changes looks good!

**Review by lsetiawan — APPROVED** [review]

**Comment by emiliom** [comment]
Thanks! Please merge as you two see fit. I'll comment on the background color choices (#3) later, likely not today.

File: webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss (hunk 2 above)
Comment (ocefpaf) [reply webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss]: I kind of wanted to preserve the original variables, but with git that does not really seem to be relevant...

Comment (lsetiawan) [reply webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss]: Ah okay. Sounds good. Thanks.

**Pull request merged**